tensorboard/webapp/app_routing/namespaced_state_reducer_helper.ts

```diff
@@ -99,10 +100,13 @@ export type NamespaceContextedState<
  *    createNamespaceContextedState(
  *        {myNamespacedState: 0},
  *        {nonNamespacedState: 'one'},
- *        (state) => {
+ *        (state, oldRoute, newRoute) => {
  *          // Perform more complex state transformations based on route kind
  *          // or the set of experiments.
- *          return {nonNamespacedState: 'one'};
+ *          if (!areSameRouteKindAndExperiments(oldRoute, newRoute)) {
+ *            return {nonNamespacedState: 'one'};
```

Comment:
This is my fault: No reason it needs to be "nonNamespaced"State I guess. That's a bit misleading. Perhaps just rename to "myState"